README.md

```diff
@@ -135,4 +135,44 @@ function findATFLazyLoadedImages() {
 
 console.log(findATFLazyLoadedImages());
 
+```
+
+### Image Info
+
+List all image resources and sort by (`name, transferSize, encodedBodySize, decodedBodySize, initiatorType`)
+
+[More Info](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming])
+
+```js
+function getImgs(sortBy) {
+  const imgs = [];
+
+  const resourceListEntries = performance.getEntriesByType("resource");
+  resourceListEntries.forEach(
+    ({
+      name,
+      transferSize,
+      encodedBodySize,
+      decodedBodySize,
+      initiatorType,
+    }) => {
+      if (initiatorType == "img") {
+        imgs.push({
+          name,
+          transferSize,
+          decodedBodySize,
+          encodedBodySize,
+        });
+      }
+    }
+  );
+
+  const imgList = imgs.sort((a, b) => {
+    return b[sortBy] - a[sortBy];
+  });
+
+  return imgList
+}
+console.log(getImgs('encodedBodySize'))
```

Comment:
We can use `console.table()` here to improve the reading

```suggestion
console.table(getImgs('encodedBodySize'))
```

## Demo

<img width="1304" alt="image" src="https://user-images.githubusercontent.com/1307927/170584929-c6ed4b8d-bde5-494b-b520-83ceaf0b99e1.png">

Reply:
Yes, great idea.